swarm/swarm.go

```diff
@@ -446,6 +449,15 @@ func (self *Swarm) Stop() error {
 		ch.Stop()
 		ch.Save()
 	}
+	if self.swap != nil {
+		self.swap.Close()
+	}
+	if self.accountingMetrics != nil {
+		self.accountingMetrics.Close()
+	}
+	if self.stateStore != nil {
```

Comment:
i dont see this assigned

Reply:
Sorry.  L#139, that `stateStore` was never being closed. So this is a fix for that, but I only half-finished that refactoring...Thanks.